extensions/wikia/GoogleAmp/GoogleAmpHelper.class.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+class GoogleAmpHelper {
+
+	/**
+	 * Checks whether AMP version is enabled for a given title.
+	 *
+	 * @param Title $title
+	 * @return bool
+	 */
+	static public function isAmpEnabled( Title $title ): bool {
+		global $wgGoogleAmpNamespaces, $wgGoogleAmpArticleBlacklist;
+
+		if ( in_array( $title->getNamespace(), $wgGoogleAmpNamespaces ) ) {
+			if ( !in_array( $title->getPrefixedDBkey(), $wgGoogleAmpArticleBlacklist ) ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Merge this if statement with the enclosing one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1066)